document-message/document-message.bal

```diff
@@ -0,0 +1,31 @@
+import ballerina/http;
+import ballerina/mime;
+
+type CsvRequest record {
+    string X_CRM_ORG;
+    string org;
+    string filename;
+    string path;
+};
+
+type ZohoResponse record {
```

Comment:
You should make this a close record since this is something we are going to return.